packages/cli-kit/src/public/node/output.ts

```diff
@@ -297,7 +297,7 @@ export function outputCompleted(content: OutputMessage, logger: Logger = console
  * @param content - The content to be output to the user.
  * @param logger - The logging function to use to output to the user.
  */
-export function outputDebug(content: OutputMessage, logger: Logger = consoleLog): void {
+export function outputDebug(content: OutputMessage, logger: Logger = consoleError): void {
```

Comment:
This is the default change. It feels safe. But we might want to make that in a different PR.